closure/goog/color/alpha.js

```diff
@@ -68,21 +68,26 @@ goog.color.alpha.hexToRgbaStyle = function(hexColor) {
 };
 
 
+function extractColor(startIndex, endIndex) {
```

Comment:
I changed to private method and added jsdoc as suggested. I don't understand why the build fails on only firefox on linux, even though the code does not seem to be browser dependent.